packages/Babylonian-Core.package/BPExampleToTest.class/instance/serializedTestName.st

```diff
@@ -0,0 +1,4 @@
+private - serializing
+serializedTestName
```

Comment:
Hm... I do not have a good overview of the relation of methods here but should that not also include the method selector?

Reply:
The test name currently is: `test<name of example in camel case>` which is what this method does. What method selector do you mean?